azure-pipelines.yml

```diff
@@ -44,6 +44,27 @@ resources:
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-14.04-cross-0cd4667-20170319080304
 
 stages:
+- stage: Create_BAR_ID_Tag
```

Comment:
I think this job should be added to the post-build stages. It can either be its own stage like this, or it could be part of the arcade promotion stage. 

As it's currently written, this is breaking the pipeline's YAML because it tries to run as part of PRs. Since it can't even queue a build, the check isn't even posted to the PR: https://dev.azure.com/dnceng/public/_build/results?buildId=1469865&view=results

Reply:
I can pull it out of PRs, however, I want to make sure that it will add the tag no matter what, so even if I add the parameter to make sure it runs even if a previous stage breaks, if the build cancels, will it still run?

Reply:
You can set the stage to run regardless of everything by adding a `condition: always` bit in the YAML. We have an example in arcade-services: https://github.com/dotnet/arcade-services/blob/ca66009f02980f8a1c04f398d1fcfab55be7465f/eng/deploy.yaml#L251-L257
It makes sense to make it its own stage then instead of making it part of the promotion stage.

Reply:
More info on that: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml